### PR TITLE
balancer pools labels

### PR DESCRIPTION
**Pull request by markusbkoch**
Adding balancer pool labels (tokens and their weights)

### CODE REVIEW

File: labels/queries/balancer_pools.sql [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    where call_success
+),
+state_with_gaps as (
+    select events.block_number, events.pool, events.token, events.denorm,
+    LEAD(cast(events.block_number as text), '1', '99999999') over (partition by events.pool, events.token order by events.block_number) as next_block_number
+    from events 
+), 
+settings as (
+    select pool, 
+    coalesce(t.symbol,'???') as symbol,
```
Comment (mewwts) [inline labels/queries/balancer_pools.sql:32]: Would it be better to inner join (i.e. only have labels with both symbols), or do you think ???/YFI is fine for now?

**Review by mewwts — APPROVED** [review]

File: labels/queries/balancer_pools.sql (hunk 1 above)
Comment (markusbkoch) [reply labels/queries/balancer_pools.sql:32]: I think it's fine for now. It helps distinguish between 2/3/4... token pools.

**Comment by mewwts** [comment]
Expecting to have this deployed in ~ a wekk @markusbkoch

**Comment by mewwts** [comment]
Running this in dev now @markusbkoch

**Pull request merged**